controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -40,6 +40,8 @@
 #include "controller_manager_msgs/srv/switch_controller.hpp"
 #include "controller_manager_msgs/srv/unload_controller.hpp"
 
+#include "std_msgs/msg/string.hpp"
```

Comment:
add this below to other includes and sort them alphabetically.

Reply:
Done